addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -3,45 +3,44 @@
+                            'o-active': storeProps.chatter.isComposerVisible and !storeProps.chatter.isComposerLog,
+                        }"
+                        t-att-disabled="storeProps.chatter.isDisabled"
+                        t-on-click="_onClickSendMessage">
+                    Send message
+                </button>
+                <button class="btn btn-link o_ChatterTopbar_buttonLogNote"
+                        type="button"
+                        t-att-class="{
+                            'o-active': storeProps.chatter.isComposerVisible and storeProps.chatter.isComposerLog,
+                        }"
+                        t-att-disabled="storeProps.chatter.isDisabled"
+                        t-on-click="_onClickLogNote">
+                    Log note
+                </button>
+            </t>
+            <t t-if="storeProps.chatter.hasActivities">
+                <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity" type="button" t-att-disabled="storeProps.chatter.isDisabled" t-on-click="_onClickScheduleActivity">
+                    <i class="fa fa-clock-o"/>
+                    Schedule activity
+                </button>
+            </t>
+            <button class="btn btn-link o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="storeProps.chatter.isDisabled" t-on-click="_onClickAttachments">
+                <i class="fa fa-paperclip"/>
+                <t t-if="storeProps.chatter.isDisabled or storeProps.areAttachmentsLoaded">
+                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="storeProps.attachmentsAmount"/>
                 </t>
-                <t t-if="storeProps.chatter.hasActivities">
-                    <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity" type="button" t-att-disabled="storeProps.chatter.isDisabled" t-on-click="_onClickScheduleActivity">
-                        <i class="fa fa-clock-o"/>
-                        Schedule activity
-                    </button>
+                <t t-else="">
+                    <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-spinner fa-spin"/>
                 </t>
-                <button class="btn btn-link o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="storeProps.chatter.isDisabled" t-on-click="_onClickAttachments">
-                    <i class="fa fa-paperclip"/>
-                    <t t-if="storeProps.chatter.isDisabled or storeProps.areAttachmentsLoaded">
-                        <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="storeProps.attachmentsAmount"/>
-                    </t>
-                    <t t-else="">
-                        <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-spinner fa-spin"/>
-                    </t>
-                </button>
-            </div>
+            </button>
+            <div class="o_ChatterTopbar_autogrow"/>
```

Comment:
just `o-autogrow` (scss rules are in owl/discuss.scss, definitely should move to owl/variables.scss)